lib/vmfloaty/utils.rb

```diff
@@ -78,27 +78,44 @@ def self.generate_os_hash(os_args)
     os_types
   end
 
+  def self.print_fqdn_for_host(service, hostname, host_data)
+    case service.type
+    when 'ABS'
+      host_data['allocated_resources'].each do |vm_name, _i|
+        puts vm_name['hostname']
+      end
```

Comment:
It's better here to build up this list of hostnames, and `puts` once.

Reply:
Oops, I misunderstood, just one second.

Reply:
There we go.